tasks/htmltasks.js

```diff
@@ -32,20 +32,6 @@ gulp.task('html:lint', () => {
     .pipe(gulpTsLint.report())
 });
 
-// Compile TypeScript to JS
-gulp.task('html:compile-src', () => {
```

Comment:
Where is this compilation being done now?

Reply:
This whole file isn't needed. There's only one file for the whole compilation. I'm gonna remove this file.